controllers/apply/form-router/submission.js

```diff
@@ -181,11 +181,9 @@ module.exports = function(
                 if (response.status !== 'OK') {
                     logger.info(`Salesforce status ${response.status}`);
                 }
+            } catch (e) {} // eslint-disable-line no-empty
 
-                next(error);
-            } catch (statusError) {
-                next(error);
-            }
+            next(error);
```

Comment:
Avoid some ambiguity around what is being caught here. Single `next` after the status check rather than duplicating.